code/src/index.css

```diff
@@ -5,9 +5,37 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100vh;
+  background:whitesmoke;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+  
 }
+
+.icecream-image {
+  width: 60px;
+  height: auto;
+  margin: 20px auto;
+  animation: rotation 1s infinite alternate ease;
+}
+
+@keyframes rotation {
+  from {
+    transform: rotate(-25deg);
+  }
+  to {
+    transform: rotate(25deg);
+  }
+}
+
```

Comment:
Love the animation of the ice-cream 😄  Is it to early for ice-cream? Makes me in the mood for it 😉